home/modules/contribute/pages/extensions-template.adoc

```diff
@@ -0,0 +1,44 @@
+= Demo of template extension
```

Comment:
(Minor) Niggle around the name "template", but that may just be the lack of familiarity with the tooling. Seems an overloaded term